config-templates/ibm/ibm-vpc-block-csi-driver/5.0/storage-class-parameters.json

```diff
@@ -29,6 +33,10 @@
         ],
         "default": "WaitForFirstConsumer",
         "required": "true",
+        "regex": "^(?:Immediate|WaitForFirstConsumer)$",
+        "min-length": 9,
```

Comment:
Is it necessary to specify length when the supported values is only two strings?

Reply:
The length limits are checked first (before regex) as a quick and easy way to reject invalid inputs.

Further, the list of `supported-values` is not used in the validation logic. It is only for informing the user of the supported values.